### PR TITLE
feat(KickAssAnime): added support for search page & replaced strings with multilingual versions

### PR DESCRIPTION
**Pull request by StrikerFRFX**
## Description 

Title

## Acknowledgements
- [x] I read the [Presence Guidelines](https://github.com/PreMiD/Presences/blob/main/.github/CONTRIBUTING.md)
- [x] I linted the code by running `yarn format`
- [x] The PR title follows the repo's [commit conventions](https://github.com/PreMiD/Presences/blob/main/.github/COMMIT_CONVENTION.md)

## Screenshots
<details>
<summary> Proof showing the creation/modification is working as expected </summary>

Discord_aJxT4BhbHP
Discord_NxBWz3OtFb
Discord_Z0w2nAVTMx

</details>

### CODE REVIEW

File: websites/K/KickAssAnime/dist/metadata.json [hunk 1]
```diff
@@ -7,14 +7,15 @@
 	"service": "KickAssAnime",
 	"description": {
 		"en": "KickAssAnime is a streaming website where you can watch TV Series, Movies & OVA in dubbed and subbed formats.",
-		"nl": "KickAssAnime is een streamingwebsite waar u tv-series, films en eicellen kunt bekijken in nagesynchroniseerde en subbedformaten."
+		"nl": "KickAssAnime is een streamingwebsite waar u tv-series, films en eicellen kunt bekijken in nagesynchroniseerde en subbedformaten.",
```
Comment (EncryptedDev) [inline websites/K/KickAssAnime/dist/metadata.json:10]: @Bas950

File: websites/K/KickAssAnime/dist/metadata.json [hunk 2]
```diff
@@ -7,14 +7,15 @@
...
+		"ga_IE": "Is suíomh Gréasáin sruthaithe é KickAssAnime inar féidir leat féachaint ar Shraith Teilifíse, ar Scannáin & OVA i bhformáidí a bhfuil dubáil agus fo-leagan orthu."
```
Comment (EncryptedDev) [inline websites/K/KickAssAnime/dist/metadata.json:11]: @callumok2004

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

File: websites/K/KickAssAnime/dist/metadata.json (hunk 1 above)
Comment (StrikerFRFX) [reply websites/K/KickAssAnime/dist/metadata.json:10]: translation is unchanged.

Comment (joerkig) [inline websites/K/KickAssAnime/dist/metadata.json:10]:
```suggestion
		"nl": "KickAssAnime is een streamingwebsite waar u tv-series, films en OVA kunt bekijken in nagesynchroniseerde en ondertitelde vorm.",
```

**Review by callumok2004 — APPROVED** [review]

**Review by EncryptedDev — APPROVED** [review]

File: websites/K/KickAssAnime/dist/metadata.json [hunk 3]
```diff
@@ -7,14 +7,15 @@
...
+		"nl": "KickAssAnime is een streamingwebsite waar u tv-series, films en OVA kunt bekijken in nagesynchroniseerde en ondertitelde vorm.",
+		"ga_IE": "Is suíomh Gréasáin sruthaithe é KickAssAnime inar féidir leat féachaint ar Shraith Teilifíse, ar Scannáin & OVA i bhformáidí a bhfuil dubáil agus fo-leagan orthu."
 	},
 	"url": [
 		"kickassanime.lol",
 		"kickassanime.ro"
 	],
 	"regExp": "([a-z0-9-]+[.])*kickassanime([.][a-z]+)+[/]",
-	"version": "1.0.21",
+	"version": "1.0.22",
```
Comment (EncryptedDev) [inline websites/K/KickAssAnime/dist/metadata.json:18]: Actually should be as below as you added support, hence minor bump
```suggestion
	"version": "1.1.0",
```

**Review by EncryptedDev — CHANGES_REQUESTED** [review]

**Review by EncryptedDev — APPROVED** [review]

**Review by Bas950 — APPROVED** [review]

**Pull request merged**